<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskMgr</class>
 <widget class="QWidget" name="TaskMgr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskMgr</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="resource_tab">
    <attribute name="title">
     <string>资源</string>
    </attribute>
    <widget class="QGroupBox" name="mem_groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>561</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>内存</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="cpu_groupBox">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>9</y>
       <width>561</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>cpu</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="disk_tab">
    <attribute name="title">
     <string>磁盘</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="progress_tab">
    <attribute name="title">
     <string>进程</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
